axelrod/tournament.py

```diff
@@ -8,9 +8,9 @@ class Tournament(object):
 
     game = Game()
 
-    def __init__(self, players, game=None, turns=200, repetitions=10):
-        """Initiate a tournmanent of players"""
-
+    def __init__(self, name, players, game=None,
```

Comment:
If it could be tested also...

On Fri, Mar 13, 2015 at 11:41 AM Owen Campbell <notifications@github.com>
wrote:

> In axelrod/tournament.py
> <https://github.com/drvinceknight/Axelrod/pull/113#discussion_r26379445>:
>
> > @@ -8,9 +8,9 @@ class Tournament(object):
> >
> >      game = Game()
> >
> > -    def __init__(self, players, game=None, turns=200, repetitions=10):
> > -        """Initiate a tournmanent of players"""
> > -
> > +    def __init__(self, name, players, game=None,
>
> Good call - on it's way
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/drvinceknight/Axelrod/pull/113/files#r26379445>.
>